application.go

```diff
@@ -34,6 +34,25 @@ type Applications struct {
 	Apps []Application `json:"apps"`
 }
 
+// IpAddress is used by Ip Per Task Funcionality https://mesosphere.github.io/marathon/docs/ip-per-task.html
```

Comment:
You're totally right about syntax (I did it at work and I don't have a proper installation in this laptop) but I'd to change struct name because it conflicts with another one defined in task.go, what do you think is a proper name, e.g. IPAddressApp???